packages/scripts/src/generateSplitDocumentation.ts

```diff
@@ -295,15 +295,15 @@ function itemExcerptText(excerpt: Excerpt, apiPackage: ApiPackage, parent?: ApiT
 			}
 
 			if (parent?.typeParameters.some((type) => type.name === token.text)) {
-				const [packageName, parentItem] = parent.canonicalReference.toString().split('!');
+				const resolvedParent = resolveCanonicalReference(parent.canonicalReference, apiPackage);
 				return {
 					text: token.text,
 					resolvedItem: {
 						kind: 'TypeParameter',
 						displayName: token.text,
 						containerKey: `${parent.containerKey}|${token.text}`,
-						uri: `${parentItem}#${token.text}`,
-						packageName: packageName?.replace('@discordjs/', ''),
+						uri: `${resolveItemURI(parent)}#${token.text}`,
```

Comment:
Not sure if this is possible, but if this branch is ever reached this would include 2 fragments on the uri

https://github.com/discordjs/discord.js/blob/09beb8a6a095eadbba33c3478d05796d6de10649/packages/scripts/src/generateSplitDocumentation.ts#L226

Reply:
That shouldn’t ever happen, since type parameters like that are only documented separately on top-level stuff like class, type alias, interface or top-level function.